src/esp/core/managedContainers/ManagedFileBasedContainer.h

```diff
@@ -163,7 +163,7 @@ class ManagedFileBasedContainer : public ManagedContainer<T, Access> {
    * extension(s) with the passed @p fileTypeExt, if it is missing.  NOTE : this
    * does not verify that file exists.
    * @param filename The original file name
-   * @param fileTypeExt The extension to use for the new filename.
+   * @param jsonTypeExt The extension to use for the new filename.
```

Comment:
This needs to be fileTypeExt - the variable name should change instead. This is not just for json anymore.